lib/kintsugi/cli.rb

```diff
@@ -88,9 +77,23 @@ def create_install_driver_subcommand
         end
 
         driver_options = options.map do |option_name, option_value|
-          prefix = option_name.length > 1 ? "--" : "-"
-          [prefix + option_name.to_s, option_value]
-        end.flatten
+          switch = option_parser.top.search(:long, option_name.to_s)
+          prefix = "--"
+          if switch.nil?
+            switch = option_parser.top.search(:short, option_name.to_s)
+            prefix = "-"
+          end
+          case switch
+          when OptionParser::Switch::NoArgument
+            [prefix + option_name.to_s]
+          when NilClass
```

Comment:
Do we want to assume that `options` are a result of parsing with the option parser? If so, invalid flags will be catched by the `parse!` method and we don't need to treat this case here.

Reply:
I don't want to have an `if else` situation that will treat `nil` as either short or long if anything changes somewhere.